README.md

```diff
@@ -275,6 +283,58 @@ connector.connect(
 )
 ``` 
 
+### Async Driver Usage
+The Cloud SQL Connector for Python currently supports the
+[asyncpg](https://magicstack.github.io/asyncpg) Postgres database driver.
+This driver leverages [asyncio](https://docs.python.org/3/library/asyncio.html)
+to improve the speed and efficiency of database connections through concurrency.
```

Comment:
nit: I would flip the phrasing of this and say "The Cloud SQL Connect is compatible with asyncio to improve the speed and efficiency of database connection through concurrency. You can use all of the non-asyncio drivers through the `Connector.connect_async` function, and the follow asyncio ones: